QueueMessageSender/QueueMessageSender/Models/RolesResultModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace QueueMessageSender.Models
+{
+    public class RolesResultModel
+    {
+        public List<RoleModel> Roles { get; set; }
```

Comment:
It is a good practice to have the same response (generic model) for all `GET List` requests. For example:
```
{
  "result": [...],
  "more": true/false
```
In current implementation you are making a lot of the same models like `RolesResultModel`, `UsersModel`, etc. It will cause also a lot of extra code on client side. Also please don't forget to use the same naming conventions.